libkbfs/block_journal_test.go

```diff
@@ -146,7 +150,7 @@ func TestBlockJournalRemoveReferences(t *testing.T) {
 
 	// Remove references.
 	liveCounts, err := j.removeReferences(
-		ctx, map[BlockID][]BlockContext{bID: {bCtx, bCtx2}}, true)
+		ctx, map[BlockID][]BlockContext{bID: {bCtx, bCtx2}}, false)
```

Comment:
Is that still a problem given my suggestion above to make `readJournal` return `blockBytes`?